mass_editing/wizard/mass_editing_wizard.py

```diff
@@ -41,172 +41,34 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
-                        'selection': field_info[field.name]['selection'],
-                    }
                 else:
-                    all_fields[field.name] = {
-                        'type': field.ttype,
-                        'string': field.field_description,
-                    }
                     all_fields["selection__" + field.name] = {
                         'type': 'selection',
                         'string': field_info[field.name]['string'],
-                        'selection': [('set', 'Set'), ('remove', 'Remove')]
+                        'selection': [('set', 'Set'),
+                                      ('remove_m2m', 'Remove')]
                     }
-                    if field.ttype == 'text':
-                        xml_group = etree.SubElement(xml_group, 'group', {
-                            'colspan': '6',
-                            'col': '6',
-                        })
-                        etree.SubElement(xml_group, 'separator', {
-                            'string': all_fields[field.name]['string'],
-                            'colspan': '6',
-                        })
-                        etree.SubElement(xml_group, 'field', {
-                            'name': "selection__" + field.name,
-                            'colspan': '6',
-                            'nolabel': '1',
-                        })
-                        etree.SubElement(xml_group, 'field', {
-                            'name': field.name,
-                            'colspan': '6',
-                            'nolabel': '1',
-                            'attrs': ("{'invisible':[('selection__" +
-                                      field.name + "','=','remove')]}"),
-                        })
-                    else:
-                        all_fields["selection__" + field.name] = {
-                            'type': 'selection',
-                            'string': field_info[field.name]['string'],
-                            'selection': [('set', 'Set'), ('remove', 'Remove')]
-                        }
-                        etree.SubElement(xml_group, 'field', {
-                            'name': "selection__" + field.name,
-                            'colspan': '2',
-                        })
-                        etree.SubElement(xml_group, 'field', {
-                            'name': field.name,
-                            'nolabel': '1',
-                            'attrs': ("{'invisible':[('selection__" +
-                                      field.name + "','=','remove')]}"),
-                            'colspan': '4',
-                        })
+                etree.SubElement(
+                    xml_group,
+                    "field",
+                    {"name": "selection__" + field.name, "colspan": "2"},
+                )
+                field_vals = {"name": field.name, "nolabel": "1", "colspan": "4"}
```

Comment:
I will test this one.

But @SimoneVagile @legalsylvain, could we just add two hook functions to get field dict options ?

```suggestion
                field_vals = self._get_field_options(field)
```

Reply:
This will allow overriding (I have the case I want to add a domain here)

Reply:
of course ! nice addition !

Reply:
I would have done the PR on Simone repo, but it's not available when creating PR

Reply:
@SimoneVagile Could you rebase ?

Reply:
And can you cherry-pick the last commit from https://github.com/acsone/server-tools/commits/10.0-imp-task-13269 ?